openstf/model/train.py

```diff
@@ -193,7 +339,7 @@ def train_model_for_specific_pj(
         model_trainer.store_model()
 
         # Save figures to disk
-        save_loc = Path(config.paths.trained_models) / str(pj["id"])
+        save_loc = Path(context.config.paths.trained_models) / str(pj["id"])
```

Comment:
It would be way more flexible if this was an argument to this function. Now you can not use it from a notebook without modifying the context first.

Reply:
added it as argument. If None is given, this is the `path_to_save`